Assets/Scripts/BlackHolePull.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class BlackHolePull : MonoBehaviour
+{
+    
+    public float gravitationalForce = 10;
+    
+    private List<Rigidbody> pulledBodies;
```

Comment:
Instead of a List of pulledBodies, you could make it a lookup storing the gameObject.name of the pulledBody or something.

Reply:
Wouldn't that be super expensive to look up each game object in the list by name on every update?

Reply:
No, you'd save it as a lookup of name to gameobject instance, so instead of a list of game objects (which could be very long depending on the number of entities being pulled), you'd have a lookup that you could interact with in constant time or loop through.

Reply:
Gotcha. Names aren't guaranteed to be unique though so I used the gameobject instance ID